puppet/zulip/files/apache/sites/zulip-sso.example

```diff
@@ -36,7 +36,7 @@ Listen 127.0.0.1:8888
 	SSLCertificateKeyFile /etc/ssl/private/zulip.key
 
 	WSGIScriptAlias / /home/zulip/deployments/current/zproject/wsgi.py
-	WSGIDaemonProcess zulip threads=5 user=zulip python-path=/home/zulip/deployments/current/
+	WSGIDaemonProcess zulip threads=5 user=zulip python-home=/home/zulip/deployments/current/zulip-py3-venv python-path=/home/zulip/deployments/current/
```

Comment:
This file, as you can see from the name, is an example.  Will the old implementation be broken when folks upgrade?

Reply:
Yeah, should this be documented in `changelog.md`?